CurrencyConverter/CurrencyConverter.Services/Services/CurrencyService.cs

```diff
@@ -69,7 +69,7 @@ public IEnumerable<Currency> GetCurrencyList()
                 return null;
             }
 
-            var rate = decimal.Round(currency2.Rate / currency1.Rate, 4);
+            var rate = currency2.Rate / currency1.Rate;
```

Comment:
Sorry, I'm confused "Improved the accuracy of conversion by rounding the rates, so if we want to display them properly we need to set how many digits to display manually." seems like you removed rounding? So was your note just phrased incorrectly, or what do you mean? I just see removal of rounding, and I can see how that would improve accuracy, but i don't see "Improved the accuracy of conversion by rounding the rates,"  :D

Reply:
Yeah, I meant removing rounding of rates, :D